keybase/platform_darwin.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package keybase
+
+import (
+	"fmt"
+	"os/user"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/kardianos/osext"
+	"github.com/keybase/go-updater"
+)
+
+// destinationPath returns the app bundle path where this executable is located.
+// Currently we only support `/Applications/Keybase.app`.
+func (c config) destinationPath() string {
```

Comment:
Yeah we can support any app bundle. Fixed and added a test to get app bundle from path instead of whitelisting.